deno.json

```diff
@@ -18,8 +18,10 @@
     "fresh_charts": "https://deno.land/x/fresh_charts@0.3.1/mod.ts",
     "lightningcss": "npm:lightningcss@^1.27.0",
     "marked": "npm:marked@^14.1.2",
+    "marked-highlight": "npm:marked-highlight@2.2.1",
     "octokit": "npm:octokit@4.0.2",
     "preact": "npm:preact@^10.24.0",
+    "prismjs": "npm:prismjs@^1.29.0",
```

Comment:
**question:** Is `marked-highlight` and `prismjs` the way we want to go? What are the bundle-sizes like?

Have you considered?:
- https://github.com/wooorm/starry-night
- https://github.com/shikijs/shiki

I have been keeping an eye on both and they seem quite good. Would you like to compare them to prism and perhaps help us make a more informed judgement on which highlighter to use? If you are satisfied with prism, we can just keep it as is, but it's always fun to try new stuff :)

Reply:
honestly i think we can stay with prism for now since we're only using it on the server to generate static content, maybe when we  start working on a playground we can reconsider?

Reply:
Sure